fineract-provider/build.gradle

```diff
@@ -88,7 +88,6 @@ dependencyManagement {
         dependency 'com.squareup.okhttp:okhttp-urlconnection:2.0.0'
         dependency 'com.google.code.gson:gson:+'
         dependency 'com.google.guava:guava:28.1-jre'
-        dependency 'org.apache.commons:commons-email:1.5'
```

Comment:
Yes, it looks but, but actually is only used in fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/service/GmailBackedPlatformEmailService.java. My  sugestion to changes this is because the mailing capabilities are served to througth org.springframework.mail.*and there are 2 more services using it and I tryed to make the changes more homogeneous